localstack-core/localstack/services/events/provider.py

```diff
@@ -1440,7 +1440,7 @@ def _process_rules(
             LOG.info(
                 json.dumps(
                     {
-                        "InfoCode": "InternalInfoEvents at matches_rule",
+                        "InfoCode": "InternalInfoEvents at matches_event",
```

Comment:
To avoid regressions in the EventStudio project, I think we should avoid this change.
Ultimately, there should be comments for every instrumentation and/or a list of instrumented places because it's currently impossible to keep track of them.

/cc @maxhoheiser @dominikschubert